### PR TITLE
[DEV-???]: add field blueprints

### PR DESCRIPTION
**Pull request by jacobmanderson**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] Jira story/task is put into PR title
- [ ] Code has been self-reviewed
- [ ] Hard-to-understand areas have been commented
- [ ] Documentation has been updated
- [ ] My changes generate no new warnings
- [ ] Unit tests have been added for base functionality and known edge cases
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: indico/types/field_blueprint.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    uid: str
+    name: str
+    version: str
+    task_type: str
+    enabled: bool
+    created_at: str
+    updated_at: str
+    created_by: int
+    updated_by: int
+    tags: "List[str]"
```
Comment (Sung96kim) [inline indico/types/field_blueprint.py]: should.. these be unquoted

**Review by Sung96kim — APPROVED** [review]
